settings/common.py

```diff
@@ -948,4 +948,4 @@
 
 
 # Ticket naming prefix
-TICKET_PREFIX = "TC-"
+os.environ.get("TICKET_PREFIX", "TC-")
```

Comment:
Sorry, I missed the assignment in my previous suggestion!
```suggestion
TICKET_PREFIX = os.environ.get("TICKET_PREFIX", "TC-")
```